src/System/Taffybar/Widget/Generic/VerticalBar.hs

```diff
@@ -103,8 +102,8 @@ liftedBarColor bc pct =
     BarConfig { barColor = c } -> return (c pct)
     BarConfigIO { barColorIO = c } -> c pct
 
-renderFrame :: Double -> BarConfig -> Int -> Int -> C.Render ()
-renderFrame pct cfg width height = do
+renderFrame_ :: Double -> BarConfig -> Int -> Int -> C.Render ()
```

Comment:
Why was the underscore added here?

Reply:
Because there's a Gtk.renderFrame already; I could have used `hiding` I guess.

Reply:
Yes, that would probably be better. Thats part of why I prefer qualified imports.